static/app/components/replays/replayContext.tsx

```diff
@@ -373,7 +373,7 @@ export function Provider({
           lineWidth: 2,
           strokeStyle: theme.purple200,
         },
-        plugins: organization.features.includes('session-replay-enable-canvas')
+        plugins: organization.features.includes('session-replay-enable-canvas-replayer')
```

Comment:
Also, this is added to those 2 python files right? That should be a separate PR so it deploys first

Reply:
yup I made a seperate PR for it!